pytorch_lightning/plugins/precision/precision_plugin.py

```diff
@@ -242,3 +242,20 @@ def teardown(self) -> None:
 
         It is the right place to release memory and free other resources.
         """
+
+    def state_dict(self) -> Dict[str, Any]:
+        """Called when saving a checkpoint, implement to generate and save precision plugin state.
```

Comment:
```suggestion
        """Called when saving a checkpoint, implement to generate precision plugin state_dict.
```
the hook specifically doesn't save it.